scripts/gitlab-rpc-docs.sh

```diff
@@ -41,12 +41,13 @@ upload_files() {
     git push --tags
 }
 
-PROJECT_DIR=$(pwd)
+RPC_TRAITS_DIR="rpc/src/v1/traits"
 
 setup_git
 cd ..
```

Comment:
This `cd ..` is not present in master, not sure what it's doing here.

Reply:
the cd does not make any sense